app/views/createRes.ejs

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+      <%=title %>
+  </title>
+</head>
+<body>
+  <% for(var i in messages) { %>
+    <div class="flash"><%=messages[i] %></div>
+  <% } %>
+  <form action="/createRes" method="post">
+    <div>
```

Comment:
A field for Event Type is missing.

Reply:
Sorry, I didn' see that the first time. After I added the entry in mongodb, everything was working.